tests/Sulu/Component/Content/Mapper/ContentMapperTest.php

```diff
@@ -3187,6 +3187,122 @@ public function testContentTypeSwitch()
         $this->mapper->save($testSiteData, 'internal-link', 'sulu_io', 'en', 1, true, $uuid);
     }
 
+    /**
+     * It should delete a node which has children with history.
+     * It should not throw an exception
```

Comment:
My point is that the DRY principle also applies to comments, and each comment should provide some value.

The comment `It should delete a node which has children with history` does not add any more value or information to a method called `testDeleteWithChildrenHistory`. If we force people to write comments like this, I am sure that this will end up in comments which will not match the test (at least after a while) and confuse more than it helps.

After having these thought I would be -1 on this.